src/relay/backend/utils.cc

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * \file relay/backend/util.cc
+ * \brief Relay backend utilities.
+ */
+
+#include "utils.h"
+
+namespace tvm {
+namespace relay {
+namespace backend {
+
+void FunctionInfo::SetWorkspaceSize(Target tgt, tvm::Integer size) {
+  (*this)->workspace_sizes.Set(tgt, size);
+}
+
+TVM_REGISTER_NODE_TYPE(FunctionInfoNode);
+TVM_REGISTER_GLOBAL("relay.backend.FunctionInfo").set_body_typed([]() { return FunctionInfo(); });
```

Comment:
how come we need this?

Reply:
I kept this to enable a developers from python to enable creation of FunctionInfo objects.

Reply:
why would someone need to do that, though?

Reply:
I think its good practice to have a constructor in python for FFI passable items.
The value I see it might be helpful to create unit tests for any future components that consume FunctionInfo.

Yes, I agree it is not obvious the value in this PR -- have a think, Its possible to take it out as its not mandatory to have this.

Reply:
so since Map is not mutable from Python, I wonder if it would make sense to also make FunctionInfo immutable in Python? I think the main question I have is: what should the API look like? it's easier to figure that out given an example of how it's used. i don't disagree with having a function like this, but since there isn't a way to use it now, someone could come and propose an API change and we wouldn't have any reasoning why to push back.

Reply:
I see; I ll drop it for now until such a time its needed.